.github/ISSUE_TEMPLATE/release.md

```diff
@@ -519,6 +520,17 @@ Follow the public disaster recovery documentation to take backups:
 - [ ] Can [take Harbor backup](https://elastisys.io/compliantkubernetes/operator-manual/disaster-recovery/#backup_1)
 - [ ] Can [take OpenSearch snapshot](https://elastisys.io/compliantkubernetes/operator-manual/disaster-recovery/#backup)
 - [ ] Can [take Velero snapshot](https://elastisys.io/compliantkubernetes/operator-manual/disaster-recovery/#backup_2)
+- [ ] Can run Rclone sync:
+
+  ```bash
+  # create rclone sync jobs for all cronjobs:
+  for cronjob in $(./bin/ck8s ops kubectl sc -n rclone get cronjobs -lapp.kubernetes.io/instance=rclone-sync -oname); do
+    ./bin/ck8s ops kubectl sc -n rclone create job --from "${cronjob}" "${cronjob/#cronjob.batch\/}"
```

Comment:
You need to write it like this `--from cronjob/cronjob-name` and then just job name

Or am I missing something here? :thinking:

```suggestion
    ./bin/ck8s ops kubectl sc -n rclone create job --from "cronjob/${cronjob}" "${cronjob}"
```

Reply:
Bashisms :smile: